assets/js/components/event/ParticipantsList.tsx

```diff
@@ -61,26 +95,36 @@ const Participant = ({ client, participant, eventChannel }: ParticipantProps) =>
     ? "Moderator"
     : participant.isPresenter
     ? "Presenter"
-    : "Praticipant";
+    : "Participant";
+
+  const isClientPariticipant: Boolean = client.email == participant.email;
 
   return (
     <div className="Participant">
       <Tooltip
-        label={`${role}${client.name == participant.name ? " (You)" : ""}`}
+        label={`${role}${isClientPariticipant ? " (You)" : ""}`}
         borderRadius="25px"
         fontSize={"1.3rem"}
         className="InfoTooltip"
       >
         {icon}
       </Tooltip>
       <p className="ParticipantText">{participant.name}</p>
+      {participant.requestPresenting && (
+        <Tooltip borderRadius="25px" fontSize={"1.3rem"} className="InfoTooltip">
+          <QuestionCircle className="ParticipantIcon" />
+        </Tooltip>
+      )}
       {client.isModerator && (
         <ModeratorMenu
           moderatorClient={client}
           eventChannel={eventChannel}
           participant={participant}
         />
       )}
+      {!client.isModerator && role == "Participant" && isClientPariticipant && (
```

Comment:
I'm a little confused. Why do we need to check 3 things: 
 - the client is not a moderator
 - the client's role is "Participant"
 - the clients is a participant (by email)

How do they differ? Shouldn't the last one imply the rest?
Is there a way to reduce it to 2 checks?

Reply:
Ok, it's bad naming. The last one is checking only if a participant object has the same email as the client object. The goal of this condition determines when the menu for asking for presenting is displayed. It makes sense only for myself when I am not a moderator and I have the role of a participant because if I am a presenter I don't have to ask for it.

Reply:
So maybe it can be moved to a separated function? Or the naming could be clearer.